### PR TITLE
guessable for missing key input

### PR DESCRIPTION
**Pull request by tksfjt1024**
guessable for missing key input

### CODE REVIEW

File: lib/embulk/guess/jsonl.rb [hunk 1]
```diff
@@ -20,8 +20,15 @@ def guess_lines(config, sample_lines)
 
         return {} if rows.size <= 3
 
-        columns = Embulk::Guess::SchemaGuess.from_hash_records(rows).map do |c|
-          column = {name: c.name, type: c.type}
+        array_of_hash = Array(rows)
```
Comment (kekekenta) [inline lib/embulk/guess/jsonl.rb]: It seems that `rows` is an Array and this Array function makes no sense.

Comment (tksfjt1024) [reply lib/embulk/guess/jsonl.rb]: fixed

File: lib/embulk/guess/jsonl.rb [hunk 2]
```diff
@@ -20,8 +20,14 @@ def guess_lines(config, sample_lines)
 
         return {} if rows.size <= 3
 
-        columns = Embulk::Guess::SchemaGuess.from_hash_records(rows).map do |c|
-          column = {name: c.name, type: c.type}
+        if rows.empty?
+          raise "SchemaGuess Can't guess schema from no records"
```
Comment (kekekenta) [inline lib/embulk/guess/jsonl.rb:25]: Is there any suitable Exception Class?

Comment (tksfjt1024) [reply lib/embulk/guess/jsonl.rb:25]: I don't know

This is copied from https://github.com/embulk/embulk/blob/master/embulk-core/src/main/ruby/embulk/guess/schema_guess.rb#L19

Comment (kekekenta) [reply lib/embulk/guess/jsonl.rb:25]: OK

**Review by kekekenta — APPROVED** [review]
LGTM

**Pull request merged**